js/init.js

```diff
@@ -1,32 +1,48 @@
 function syncState() {
-  chrome.storage.sync.get('classList', function(result) {
+  let board_id = location.pathname.split('/')[2];
+  let storageKey = 'classList_' + board_id;
+
+  chrome.storage.sync.get(storageKey, function(result) {
     let board = document.getElementById('board');
 
-    if (result.classList) {
-      board.classList.add(result.classList);
+    if (result[storageKey]) {
+      board.classList.add(result[storageKey]);
     }
   });
 }
 
+chrome.storage.onChanged.addListener(function (changes, namespace) {
+  let board = document.getElementById('board');
+  let board_id = location.pathname.split('/')[2];
+  let storageKey = 'classList_' + board_id;
```

Comment:
Since the storageKey construction is repeated in a few places, can you extract it to a function?